content/docs/using-regolith/install.ja.md

```diff
@@ -60,15 +60,15 @@ Regolithはシステムパッケージとしてインストールすることが
 1. ローカルの`apt`にRegolithの公開鍵を登録する。
 
    ```console
-   wget -qO - https://regolith-desktop.org/regolith.key | \
+   wget -qO - http://archive.regolith-desktop.com/regolith.key | \
    gpg --dearmor | sudo tee /usr/share/keyrings/regolith-archive-keyring.gpg > /dev/null
    ```
 
 1. ローカルの`apt`にリポジトリのURLを追加する。
 
    ```console
    echo deb "[arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] \
-   https://regolith-desktop.org/release-3_2-ubuntu-jammy-amd64 jammy main" | \
+   http://archive.regolith-desktop.com/ubuntu/stable jammy v3.2" | \
```

Comment:
as discussed elsewhere let's move these URLS to use the `https` scheme and somehow mention that `http` is also usable.

Reply:
Should be resolved by [b236a37](https://github.com/regolith-linux/regolith-desktop-website/pull/61/commits/b236a37b8b41d4317e66ea3c9ce08cb1b43cb718)